tests/Backend/Workspaces/WorkspacesSnowflakeTest.php

```diff
@@ -867,17 +867,35 @@ public function testTableLoadAsView(): void
             'languages',
             new CsvFile($importFile)
         );
+        $aliasId = $this->_client->createAliasTable($bucketId, $tableId, 'languages-alias');
 
         $options = [
             'input' => [
                 [
-                    'source' => $tableId,
+                    'source' => $aliasId,
                     'destination' => 'languages',
                     'useView' => true,
                 ],
             ],
         ];
 
+        try {
+            $workspaces->loadWorkspaceData($workspace['id'], $options);
+            $this->fail('Must throw exception as load alias as view is not supported');
+        } catch (ClientException $e) {
+            $this->assertSame('workspace.loadRequestLogicalException', $e->getStringCode());
+            $this->assertSame(
+                'View load is not supported, only table can be loaded using views, alias of table supplied. Use read-only storage instead or copy input mapping if supported.',
+                $e->getMessage()
+            );
+        } catch (Throwable $e) {
```

Comment:
todle tady potrebujeme? jinde to nedelame a ve vysledku to chcipne uplne stejne ne?

Reply:
Toho sem si všiml, že to děláme na hovno, pokud ti tam vyskočí jiná exception tak ten test nefailne správně.

Reply:
no kdyz tam mas `catch (ClientException $e)` (a k tomu idealne nejaky assert na message nebo tak neco), tak je to v cajku ne? 

kdyz to bude bez toho `catch (Throwable $e)` a vypadne tam neco jineho, tak to vesele chcipne, nebo se mylim?

Reply:
Ty vole su dement :D